docs/admin/system-information.rst

```diff
@@ -1729,6 +1729,12 @@ After you reindexed all tables, this cluster check will pass.
 License check
 .............
 
+
+.. NOTE::
+
+   This check is obsolete with CrateDB 4.5
```

Comment:
how about: "This check was removed in CrateDB version 4.5."

ideally, this would link through to the release notes with a ref, but that isn't possible until the 4.5 release notes file is created

Reply:
I've changed it